pkg/descheduler/apis/config/validation/validation_loadaware_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2023 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"testing"
+
+	deschedulerconfig "github.com/koordinator-sh/koordinator/pkg/descheduler/apis/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateLowLoadUtilizationArgs_NumerOfNodes(t *testing.T) {
```

Comment:
These codes have nothing to do with the PR itself. It is recommended to submit it as a new PR independently to ensure that each PR only does one thing.

Reply:
I've forgotten when it was changed, so I won't dismantle it.

Reply:
ok.